### DISCUSSION TITLE
SonixTablet:  How to record live sequences, and then generate ultrasound volume from sequences?

### DISCUSSION THREAD

**Discussion by SenonETS** [PlusToolkit/PlusLib#819] [Q&A]
Hello,

Just know how to collect ultrasound volume using the VolumeReconstruction module in the slicer, however, it seems that the volume has to be with cubic shape controlled by ROI.

Could anyone help on how to record sequences (e.g., 10 scans in a sequence) in 3D, like collecting a 3D video (from the first scan to the 10th) and then show all the scans (10 scans in total) at the same time?

And how to generate a volume file using the recorded sequence?

**Comment by lassoan** [comment]
What would you like to scan? Vertebrae?

**Reply by SenonETS** [reply]
Yes. Specifically, the whole thoracic and lumbar regions in one sequence.

**Reply by lassoan** [reply]
We typically do a quick scout scan blindly (without seeing the reconstructed volume live), set the region of interest based on this quickly acquired, low-quality volume, and then start the high-quality scan with live image reconstruction. If you place your reference marker consistently then you even may skip the scout scan and work with a fixed ROI (specified relative to the reference marker).

See a detailed demo video here: https://www.youtube.com/watch?v=l0BcW8c9CnI

**Reply by SenonETS** [reply]
Yes, I have watched that video before, a very impressive demo!

However, my study is more related to sequence analysis, instead of vertebral column reconstruction.

Could you please help me with some sequence recording tutorials, and sequence to volume ideas?

**Reply by lassoan** [reply]
Can you describe what your end goal is? We can then give advice on how to achieve it.

**Reply by SenonETS** [reply]
The goal is to display a sequence of 2D scans at the same time, just like [this image](https://ibb.co/ZdTTwx1)

**Reply by lassoan** [reply]
What we describe above generates exactly this kind of volume.

**Reply by SenonETS** [reply]
I am sorry that my description was not clear. I think there might be a misunderstanding around the so-called "volume".

What I would like is firstly to record sequence, e.g. 10 pieces of 2D scans, instead of a cube.

Secondly, I would like to display the 10 pieces of scans like [this image](https://ibb.co/ZdTTwx1), instead of three analogy cross-sections from a black cube.

Thirdly, I would like to save them like saving a cubic volume, but instead of all the other voxels being black, I wish them to be transparent.

Please kindly help with some tutorials link for a sequence recording, if the other two objectives are impossible in Slicer.

**Reply by lassoan** [reply]
What is the clinical task? Scoliosis assessment, pedicle screw placement planning, ablation planning, ...?

**Reply by SenonETS** [reply]
Vertebral landmark recognition using ultrasound. 
More practically, I just would like to learn how to record sequences using Slicer.
